tests/installation/first_boot.pm

```diff
@@ -18,12 +18,16 @@ use warnings;
 use base 'bootbasetest';
 use testapi;
 use x11utils 'turn_off_plasma_tooltips';
+use utils 'handle_gnome_memory_ge_4g';
 
 sub run {
     shift->wait_boot_past_bootloader;
     # This only works with generic-desktop. In the opensuse-welcome case,
     # the opensuse-welcome module will handle it instead.
     turn_off_plasma_tooltips if match_has_tag('generic-desktop');
+
+    # https://progress.opensuse.org/issues/153808
+    handle_gnome_memory_ge_4g();
```

Comment:
This can be moved to the beginning of the `run { }` before anything else

Reply:
I have some concern about putting it at first, we may miss some fatal issues at boot phase if any :)